cmd/cat/cat.go

```diff
@@ -0,0 +1,49 @@
+// Based on Plan9 cat
+
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func cat(f *os.File, name string) {
+	var (
+		buf [8192]byte
+	)
+
+	for {
+		n, err := f.Read(buf[:])
+
+		if err != nil {
+			log.Fatal("error reading %s: %s", name, err)
```

Comment:
@tiago4orion we are no handling EOF, should we? Because in case of success the error will be EOF and a log.Fatal() will be executed. We could move **if n <= 0 {}** to just after **n, err := f.Read(buf[:])** or use io.EOF to catch the error. I'm talking bullshit? What do you think?